### ISSUE TITLE
why add TimeDistributed(Dense(nh)) between BiLSTM in network_keras.py 

### ISSUE THREAD

**Issue by piaobuliao** [chineseocr/chineseocr#392]
why add TimeDistributed(Dense(nh)) between BiLSTM in network_keras.py 。
in https://github.com/meijieru/crnn.pytorch/blob/master/models/crnn.py，it look like this:
BidirectionalLSTM(512, nh, nh),
BidirectionalLSTM(nh, nh, nclass))

**Comment by fanhenghui** [comment]
看你昵称像中国人，拽什么英文啊

**Comment by piaobuliao** [comment]
@fanhenghui  brother,I'm sorry

**Comment by fanhenghui** [comment]
还装

**Comment by wenlihaoyu** [comment]
TimeDistributed 只是keras的一个封装器而已，具体用法可参考相关文档。@fanhenghui  请不要讨论和技术无关的话题

**Comment by piaobuliao** [comment]
@wenlihaoyu TimeDistributed的功能我知道，我的意思为啥在network_keras.py里要在  Bidirectional(LSTM(  .. ) 之间加 Dense(nh)，看你的在network_torch.py里的是BidirectionalLSTM直接连BidirectionalLSTM

       x = Bidirectional(LSTM(nh, return_sequences=True, use_bias=True,
                               recurrent_activation='sigmoid'))(x)
        x = TimeDistributed(Dense(nh))(x)
        x = Bidirectional(LSTM(nh, return_sequences=True, use_bias=True,
                               recurrent_activation='sigmoid'))(x)
